tests/jaxite/end_to_end/test.bzl

```diff
@@ -1,10 +1,11 @@
 """A macro providing an end-to-end test for jaxite codegen."""
 
+load("@heir//tools:heir-opt.bzl", "heir_opt")
 load("@heir//tools:heir-translate.bzl", "heir_translate")
 load("@rules_python//python:py_library.bzl", "py_library")
 load("@rules_python//python:py_test.bzl", "py_test")
 
-def jaxite_end_to_end_test(name, mlir_src, test_src, tags = [], deps = [], **kwargs):
+def jaxite_end_to_end_test(name, mlir_src, test_src, heir_opt_flags = "", tags = [], deps = [], **kwargs):
```

Comment:
buildifier is complaining that `heir_opt_flags` isn't documented in CI!

Reply:
Done!